content/essentials/memos.mdx

```diff
@@ -102,7 +107,20 @@ createEffect(() => {
 });
 ```
 
-In the revised approach, the `isEven` memo will purely compute based on whether `count` is even *without* side effects. 
-In using the `createEffect` function, the side effects are appropriately handled. 
+In this updated approach, the `isEven` memo will compute based on whether `count` is even *without* side effects.
+However, to minimize unnecessary updates that occur as a result of a change, it is better practice to convert any unnecessary effects into memos.
+This can help to minimize unnecessary updates in other parts of the application.
```

Comment:
```suggestion
This helps minimize unnecessary updates in other parts of the application, because it lets Solid understand the reactive dependencies and thus optimize evaluation order.
```